packages/menu/src/MenuItem.tsx

```diff
@@ -192,6 +191,7 @@ const MenuItem = React.forwardRef(
           {...anchorProps}
           {...rest}
           className={cx(
+            MenuItemContainerClassName,
```

Comment:
nit but should this be `menuItemContainerClassName`

Reply:
oh good catch